hazenlib/ACRObject.py

```diff
@@ -124,7 +124,7 @@ def rotate_images(self):
 
         return skimage.transform.rotate(self.images, self.rot_angle, resize=False, preserve_range=True)
 
-    def find_phantom_center(self):
+    def find_phantom_center(self, img):
```

Comment:
this is great, will have to update the docstring for the function

Reply:
done